src/com/owncloud/android/services/OperationsService.java

```diff
@@ -521,7 +521,6 @@ private void nextOperation() {
                     }
                 }
                 
-                //sendBroadcastOperationFinished(mLastTarget, mCurrentOperation, result);
```

Comment:
sendBroadcastOperationFinished method is not currently being used, I guess you decided to keep it since it could be useful in the future.

Reply:
I kept serveral methode, because it was logic to have them. However if you say we don't need those as well lets remove them.

Reply:
Ok, I would keep this method since is logic to have it.